### PR TITLE
Update progress indicator based on task status

### PR DESCRIPTION
**Pull request by aestene**

### CODE REVIEW

File: frontend/src/components/MissionOverview/MissionTagDisplay.tsx [hunk 1]
```diff
@@ -1,14 +1,38 @@
...
+export function MissionProgressDisplay({ tasks }: TaskProps) {
+    const [completedTasks, setCompletedTasks] = useState<number>(0)
+    useEffect(() => {
+        setCompletedTasks(countCompletedTasks(tasks))
+    }, [tasks])
+
+    const countCompletedTasks = (tasks: IsarTask[]) => {
+        var counter = 0
+        tasks.forEach((task) => {
+            console.log(task.taskStatus)
```
Comment (eivindsjovold) [inline frontend/src/components/MissionOverview/MissionTagDisplay.tsx:23]: Remove console.log

File: frontend/src/components/MissionOverview/MissionTagDisplay.tsx [hunk 2]
```diff
@@ -1,14 +1,38 @@
...
+
+export function MissionProgressDisplay({ tasks }: TaskProps) {
+    const [completedTasks, setCompletedTasks] = useState<number>(0)
+    useEffect(() => {
+        setCompletedTasks(countCompletedTasks(tasks))
+    }, [tasks])
+
+    const countCompletedTasks = (tasks: IsarTask[]) => {
+        var counter = 0
+        tasks.forEach((task) => {
```
Comment (eivindsjovold) [inline frontend/src/components/MissionOverview/MissionTagDisplay.tsx:22]: Suggestion: Replace tasks.forEach() with:
let counter = 0
tasks.map((task) => IsarTaskStatus.isComplete(task.taskStatus) ? counter += 1 : counter += 0)

**Review by eivindsjovold — CHANGES_REQUESTED** [review]

File: frontend/src/components/MissionOverview/MissionTagDisplay.tsx [hunk 3]
```diff
@@ -1,14 +1,38 @@
...
+            console.log(task.taskStatus)
+            if (IsarTaskStatus.isComplete(task.taskStatus)) {
+                counter = counter + 1
+            }
+        })
```
Comment (aestene) [inline frontend/src/components/MissionOverview/MissionTagDisplay.tsx:27]:
```suggestion
        var counter = 0
        tasks.map((task) => IsarTaskStatus.isComplete(task.taskStatus) ? 
counter += 1 : counter += 0)
```

**Review by eivindsjovold — APPROVED** [review]
LGTM

**Pull request merged**